ProjectSettings/ProjectVersion.txt

```diff
@@ -1 +1,2 @@
-m_EditorVersion: 2018.4.22f1
+m_EditorVersion: 2019.2.19f1
```

Comment:
Careful working with 2019 when making PRs, as this repo is setup with 2018.4 for now, at least until 2019.3 is the official version recommended for MRTK.

Reply:
Reverted.